server/@types/crm4/index.d.ts

```diff
@@ -0,0 +1,130 @@
+type Firm = {
+  urn: string
+  firmAddress: string
+  firmName: string
+  firmPhone: string
+  firmSupplierName: string
+  firmSupplierNo: string
+  firmContactName: string
+  firmSolicitorName: string
+}
+
+type SolicitorDetails = {
+  solicitorName: string
+  solicitorReference: string
+}
+
+type ClientDetails = {
+  ufn: string
+  maatNumber: string
+  firstName: string
+  surname: string
+  dateOfBirth: string
+  prisonLaw: string
+}
+
+type ProceedingDetails = {
+  isClientDetained: string
+  detainedDetails: string
+  courtType: string
+  psychiatricLiaison: string
+  psychiatricDetails: string
+  mainOffence: string
+  actualPlea: string
+  dateOfHearing: string
+}
+
+type CaseDetails = {
+  isPOCA: string
+  Firm: Firm
+  SolicitorDetails: SolicitorDetails
+  ClientDetails: ClientDetails
+  ProceedingDetails: ProceedingDetails
+}
+
+type Details = {
+  expenditureType: string
+  priorAuthority: string
+  expertName: string
+  companyName: string
+  statusExpert: string
+  postCodeExpert: string
+}
+
+type Preparation = {
+  hours: string
+  hourlyRate: number
+  total: number
+}
+
+type Travel = {
+  hours: string
+  rate: number
+  total: number
+}
+
+type AdditionalExpenditure = {
+  description: string
+  justification: string
+  quantity: number
+  rate: number
+  total: number
+}
+
+type ExpenditureDetails = {
+  Details: Details
+  Preparation: Preparation
+  AdditionalExpenditure: Array<AdditionalExpenditure>
+  Travel: Travel
+  Authority: number
+}
+
+type Quote = {
+  companyName: string
+  expertName: string
+  contactPhone: string
+  costBasis: string
+  preparationHours: number
+  hourlyRate: number
+  // typos to be fixed in BE code
```

Comment:
Do we need a ticket related to these typos?

Reply:
In the past, a ticket has been raised on the BE first. Then once the changes are available they are made in the FE.
At the moment the typos are negligible, as the story for the display configuration has not been implemented yet.

Reply:
Raising a ticket for the BE now.